internal/expr/typeinfo_test.go

```diff
@@ -3,14 +3,10 @@ package expr
 import (
 	"reflect"
 	"sync"
-	"testing"
 
 	. "gopkg.in/check.v1"
 )
 
-// Hook up gocheck into the "go test" runner.
-func TestInternal(t *testing.T) { TestingT(t) }
```

Comment:
I assume this has been replaced elsewhere ;)

Reply:
Yes, it turns out you only need one for both the internal and external testing, even though they're in different packages (`expr` vs `expr_test`). All the test results were appearing twice.